src/App.vue

```diff
@@ -2,37 +2,91 @@
   <v-app>
     <v-toolbar app>
       <v-toolbar-title class="headline text-uppercase">
-        <span>Vuetify</span>
-        <span class="font-weight-light">MATERIAL DESIGN</span>
+        <span>Posts</span>
       </v-toolbar-title>
       <v-spacer />
-      <v-btn
-        flat
-        href="https://github.com/vuetifyjs/vuetify/releases/latest"
-        target="_blank"
-      >
-        <span class="mr-2">Latest Release</span>
-      </v-btn>
     </v-toolbar>
-
     <v-content>
-      <HelloWorld />
+      <v-card min-height="100%">
+        <v-container
+          grid-list-lg
+        >
+          <v-layout>
+            <v-flex>
+              <v-text-field
+                v-model="searchText"
+                label="Post Id"
+                single-line
+                outline
+              />           
+            </v-flex>
+            <v-flex> 
+              <v-btn
+                large 
+                color="primary"
+                @click="loadPost"
+              >
+                Load
+              </v-btn>
+            </v-flex>
+          </v-layout>
+          <post
+            v-for="(post, index) in posts"
+            :key="index"
+            :post="post"
+          />
+        </v-container>
+      </v-card>
     </v-content>
   </v-app>
 </template>
 
 <script>
-import HelloWorld from './components/HelloWorld'
+import axios from "axios";
+import Post from "./components/Post";
 
 export default {
   name: 'App',
   components: {
-    HelloWorld
+    Post,
   },
   data () {
     return {
-      //
+      searchText: '',
+      posts: [],
     }
-  }
+  },
+  mounted () {
+    var vm =this;
+    axios.get("https://jsonplaceholder.typicode.com/posts")
```

Comment:
We don't need a list when the page is mounted.
Only need to get the page once the user wants to load a post.